test/fixtures/tsx/functions/func1.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
Are we getting anything from actually including React in a fixture and committing its source to version control? It's fine to do it if we think it will make the test more accurate, but it feels that in this case we only want to check if a global is defined and not even using the module itself (which means it's probably getting tree-shaken)?

Reply:
If you take a look at the sources, you'll see that it's two placeholders and there's no real React in there. Happy to rename to a more telling name, if that makes it less confusing.

Reply:
Oh, I see! GitHub hides the source of those files by default and I had assumed they were the real thing. Sorry, ignore me!